Solutions/Corvus.Testing.AzureFunctions.Xunit.Demo/AzureFunctionFixture.cs

```diff
@@ -0,0 +1,40 @@
+// <copyright file="AzureFunctionFixture.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Testing.AzureFunctions.Xunit.Demo
+{
+    using System;
+    using System.Threading.Tasks;
+    using global::Xunit;
```

Comment:
What makes the `global::` prefix necessary here?

Reply:
Ah, the fact that we have `Xunit` in the namespace name, and that the `using` is nested within that namespace. It's not necessary when imported outside the namespace (into `global` scope, I guess, so makes sense), but that breaks IDE0065 and SA1200.